Makefile

```diff
@@ -180,17 +179,11 @@ image-codegen:
 	$(QUIET)echo "Push like this when ready:"
 	$(QUIET)echo "${CONTAINER_ENGINE} push cilium/tetragon-codegen:$(DOCKER_IMAGE_TAG)"
 
-.PHONY: tools-install tools-clean clang-install
-tools-install: clang-install
-tools-clean:
-	rm -rf $(CLANG_INSTALL_DIR)
-
-clang-install:
-	$(eval id=$(shell $(CONTAINER_ENGINE) create $(CLANG_IMAGE)))
-	mkdir -p $(CLANG_INSTALL_DIR)
-	$(CONTAINER_ENGINE) cp ${id}:/usr/local/bin/clang-11 $(CLANG_INSTALL_DIR)/clang
-	$(CONTAINER_ENGINE) cp ${id}:/usr/local/bin/llc $(CLANG_INSTALL_DIR)/llc
-	$(CONTAINER_ENGINE) stop ${id}
+.PHONY: image-clang
+image-clang:
+	$(CONTAINER_ENGINE) build -f Dockerfile.clang --build-arg VERSION=1:13.0.1-2ubuntu2 -t "cilium/clang:${DOCKER_IMAGE_TAG}" .
```

Comment:
reading https://docs.cilium.io/en/latest/contributing/development/images/, how about we use cilium/tetragon-clang for the image name? I think it would be less confusing for folk

Reply:
@willfindlay please address this comment if needed with follow up.